package/MDAnalysis/coordinates/TNG.py

```diff
@@ -0,0 +1,507 @@
+algorithms and unlike the compressed XTC format can also store velocities and
+forces in addition to positions.
+
+The classes in this module are based on the `pytng`_ package for reading TNG
+files. The reader is directed to the `pytng documentation`_ for further reading
+about how pytng works under the hood. Pytng is an optional dependency and must
+be installed to use this Reader. Use of the reader without pytng installed will
+raise an `ImportError`. The variable `HAS_PYTNG` indicates whether this module
+has pytng availble.
+
+In addition to particle-dependent trajectory information like positions,
+forces and velocities, the TNG format can store trajectory metadata and
+other arbitrary time dependent data. Additional information can range from
+the virial and pressure components to the molecular topology of the system.
+This is enabled by a block based system in which binary flags indicate the
+presence or absence of various data blocks. The structure of a TNG file is
+provided  in the `TNG specification`_. The TNG paper :cite:p:`Lundborg2014` and
+the `pytng documentation`_ are also good resources. The user is encouraged to
+read the full list of `TNG blocks`_ to understand the full power of the TNG
+format.
+
+
+
+Current Limitations
+-------------------
+Currently there is only a Reader for TNG files and no Writer. This will depend
+on upstream changes in pytng. Additionally, it is not currently possible to
+read the molecular topology from a TNG file.
+
+There are also some limitations to reading TNG files with pytng.
+Currently we do not allow data to be read *off stride*. In essence this
+means that all of the critical trajectory data (positions, box, velocities
+(if present), forces (if present)) must share the same stride in trajectory
+integrator steps. These critical blocks in the TNG file are henceforth called
+*special blocks*. Optional blocks (all blocks that are not special blocks)
+will not be read if they do not share an integrator step with, or are not
+divisible by the shared integrator step of the special blocks.
+
+
+References
+----------
+
+.. bibliography::
+    :filter: False
+    :style: MDA
+
+    Lundborg2014
+
+
+.. Links
+
+.. _GROMACS:
+    https://www.gromacs.org/
+.. _pytng:
+    https://github.com/MDAnalysis/pytng
+.. _pytng documentation:
+    https://www.mdanalysis.org/pytng/
+.. _TNG blocks:
+    https://www.mdanalysis.org/pytng/documentation_pages/Blocks.html
+.. _TNG specification:
+    https://gitlab.com/hugomacdermott/tng/-/blob/master/Trajectoryformatspecification.mk
+
+"""
+
+import warnings
+from typing import List, Optional
+
+import MDAnalysis as mda
+import numpy as np
+from MDAnalysis.coordinates import base
+from MDAnalysis.coordinates.timestep import Timestep
+from MDAnalysis.lib.mdamath import triclinic_box
+
+from ..due import Doi, due
+
+try:
+    import pytng
+except ImportError:
+    #: Indicates whether pytng is found.
```

Comment:
readthedocs is having a field day here, can you confirm this renders fine locally @hmacdope ? https://mdanalysis--3765.org.readthedocs.build/en/3765/documentation_pages/coordinates/TNG.html

Reply:
Seems to be ok locally, what specifically where you looking at?

Reply:
This seems out of place https://mdanalysis--3765.org.readthedocs.build/en/3765/documentation_pages/coordinates/TNG.html#MDAnalysis.coordinates.TNG.HAS_PYTNG

There's also some weird placement of the author list.

Reply:
Removing the comments of the form `#: ` fixed it.